_data/tools.yml

```diff
@@ -725,6 +725,24 @@
       url: https://www.microsoft.com/sql-server
     - urltext: SQL Server connector documentation
       url: https://trino.io/docs/current/connector/sqlserver.html
+- name: Microstrategy
+  anchor: microstrategy
+  category: client
+  logo: /assets/images/logos/microstrategy.png
+  logosmall: /assets/images/logos/microstrategy-small.png
+  description: |
+    MicroStrategy is the largest independent publicly traded analytics and
```

Comment:
Dont talk about the company and their status .. leave this to be about the tool

Reply:
So cut this and go

MicroStrategy is a business intelligence tool that allows you..